crates/db_schema/src/lib.rs

```diff
@@ -175,6 +175,7 @@ pub enum SearchType {
   Communities,
   Users,
   Url,
```

Comment:
@dessalines What do you think about renaming Url to PostUrl?

Reply:
I'd be fine with that.